frontend/src/slices/auth/auth.slice.ts

```diff
@@ -11,6 +12,7 @@ import {
 import { getCurrent, signIn, signUp } from './actions.js';
 
 type State = {
+  error?: HttpError;
```

Comment:
```suggestion
  error: HttpError | null;
```
Required properties are more preferable than optional